burn-ndarray/tests/no_std_checks.rs

```diff
@@ -0,0 +1,94 @@
+// This module contains tests to verify substituded functions are the same with no_std and std
```

Comment:
I'm not sure we should keep this test. It may have been usefull to debug, but it essentially tests other crates.

Reply:
OK. I'll run the clippy and fmt. Also I'll remove the tests.